lib/flutter_map_tappable_polyline.dart

```diff
@@ -16,22 +16,33 @@ class TappablePolylineMapPlugin extends MapPlugin {
   }
 }
 
+/// The options allowing tappable polyline tweaks
 class TappablePolylineLayerOptions extends PolylineLayerOptions {
+  /// The list of [TaggedPolyline] which could be tapped
   @override
   final List<TaggedPolyline> polylines;
+
+  /// The tolerated distance between pointer and user tap to trigger the [onTap] callback
   final double pointerDistanceTolerance;
+
+  /// The callback to call when tapping the polyline
   Function onTap = (TaggedPolyline polyline) {};
 
+  /// The ability to render only polylines in current view bounds
+  final bool polylineCulling;
+
   TappablePolylineLayerOptions(
       {this.polylines = const [],
       rebuild,
       this.onTap,
       this.pointerDistanceTolerance = 15,
-      polylineCulling = false})
-      : super(rebuild: rebuild, polylineCulling: polylineCulling);
```

Comment:
Why this removal?

Reply:
This isn't a removal. `polylineCulling` is now a final property of `TappablePolylineLayerOptions`

Reply:
You have to pass it to super

Reply:
Changes done